integration/bundle/apps_test.go

```diff
@@ -98,6 +98,24 @@ func TestDeployBundleWithApp(t *testing.T) {
   - run
 env:
   - name: JOB_ID
+    value: "%d"`, job.JobId))
+
+	// Redeploy bundle with changed config env for app and confirm it's updated in app.yaml
+	deployBundleWithArgs(t, ctx, root, `--var="env_var_name=ANOTHER_JOB_ID"`, "--force-lock", "--auto-approve")
+	reader, err = wt.W.Workspace.Download(ctx, pathToAppYml)
+	require.NoError(t, err)
+
+	data, err = io.ReadAll(reader)
+	require.NoError(t, err)
+
+	content = string(data)
+	require.Contains(t, content, fmt.Sprintf(`command:
+  - flask
+  - --app
+  - app
+  - run
+env:
+  - name: ANOTHER_JOB_ID
```

Comment:
FYI, non-blocking: you can also use golden files approach for golang integration tests, see example here https://github.com/databricks/cli/blob/27eb0c40725aa978191ce3fe43eb07993fd6cfe4/integration/bundle/init_default_python_test.go#L112

Reply:
It's a bit weird to change the env var name. Alternatively, you could add a second env var and change the value for something that is more likely to reflect reality.

The outcome is the same, so not blocking.